src/main/java/com/suse/saltstack/netapi/client/SaltStackClient.java

```diff
@@ -177,4 +178,56 @@ public SaltStackJob minions(String target, String function, List<String> args,
         // They return a list of tokens here, we take the first
         return result.getJobs().get(0);
     }
+    /**
+     * Generic interface to start any execution command bypassing normal session handling.
+     *
+     * POST /run
+     *
+     * @param username the username
+     * @param password the password
+     * @param eauth the eauth type
+     * @param client the client
+     * @param target the target
+     * @param function the function to execute
+     * @param args list of non-keyword arguments
+     * @param kwargs map containing keyword arguments
+     * @return object representing the scheduled job
+     * @throws SaltStackException if anything goes wrong
+     */
+    public SaltStackRunResults run(String username, String password, String eauth,
+        String client, String target, String function, List<String> args, Map<String,
+        String> kwargs) throws SaltStackException {
```

Comment:
Wrapped lines should be indented twice as much as the normal indentation = 8 spaces. Sorry, these rules are not yet written down anywhere in this repo.